Automattic/simplenote-macos
# New Metrics Screen: Mark 1

### Details:
In this PR we're implementing a brand new Metrics Window, in charge of displaying the Note's Details:

@aerych Last one of the week, promise!! (Thank you!!)

Ref. #458

### Test: Single Note
1. Launch Simplenote
2. Select a single note
3. Click over the top right `( i )` icon

- [x] Verify the new Metrics UI is presented from the `( i )` icon
- [x] Verify that switching to Light or Dark theme refreshes the Metrics UI
- [x] Verify that logging out while the Metrics UI is OnScreen, doesn't cause the Metrics Ui to be visible right after logging in again

**Notes:**
- In a Follow Up PR we're bringing realtime metrics
- And **in anoother** Follow Up PR we'll be normalizing the Popover UX all over

### Test: Multiple Notes
1. Launch Simplenote
2. Control+click multiple notes on the Notes List
3. Click over the top right `( i )` icon

- [x] Verify the Date fields display a `-`
- [x] Verify the Characters / Words fields display the sum of those values, for the selected notes

### Release
`RELEASE-NOTES.txt` was updated in f90458e with:
 
> New Note Metrics Interface


### Screenshots:
<img width="313" alt="Screen Shot 2020-06-30 at 4 46 54 PM" src="https://user-images.githubusercontent.com/1195260/86170316-5558f780-baf1-11ea-9242-335805c42ef9.png">
<img width="313" alt="Screen Shot 2020-06-30 at 4 47 45 PM" src="https://user-images.githubusercontent.com/1195260/86170394-715c9900-baf1-11ea-9075-313a8654bb8c.png">
